style.css

```diff
@@ -8,10 +8,67 @@
   padding: 0 24px;
```

Comment:
[OPTIONAL] Please remove commented code on your project files. It will keep your codebase neat and professional